### PR TITLE
Hocs 2619 serenity automation tests

### PR DESCRIPTION
**Pull request by Paulcr-ho**
Setting header added as per plugins/s3 documentation.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -53,12 +53,13 @@ pipeline:
...
     image: plugins/s3
     secrets: [ s3_bucket_name, aws_access_key_id, aws_secret_access_key ]
-    region: [ eu-west-2 ]
-    bucket: cs-test-s3
-    access_key: ${AWS_ACCESS_KEY_ID}
-    secret_key: ${AWS_SECRET_ACCESS_KEY}
-    source: target/site/serenity/**/*
-    target: /reports/$${DRONE_BUILD_NUMBER}/
+    settings:
+      region: [ eu-west-2 ]
```
Comment (zuzak) [inline .drone.yml:57]: Is this supposed to be an array?

**Review by zuzak — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (Paulcr-ho) [reply .drone.yml:57]: Probably not. Please create a suggestion and I will commit.

Comment (zuzak) [inline .drone.yml:57]:
```suggestion
      region: eu-west-2
```

Comment (Paulcr-ho) [reply .drone.yml]: Suggestion commited.

**Review by zuzak — APPROVED** [review]

**Pull request merged**